services/core/shared/database/knex_migrations/transactions.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const config = require('../../../config');
+
+const tableName = config.db.collections.transactions.name;
+
+exports.up = knex => knex.schema
+    .createTable(tableName, table => {
+        // Indexed properties
+        table.string('id').primary();
+        table.integer('height').notNullable().index();
```

Comment:
In case we plan to follow the 3NF strictly, `height` can be resolved from `blocks` index based on the `blockId`. Therefore, we wouldn't need to duplicate it here in transactions.

This is an idea for an open discussion. However, I do not see any issues with the current approach.

Reply:
I referred to the document `Lisk Service SDK v5 Update Specification` and it says to indexed `height` and `timestamp` while indexing transactions. Whats your thought @MichalTuleja ?